src/components/domain/modals/TaskCardModal.tsx

```diff
@@ -0,0 +1,284 @@
+          comment.id === id ? { ...comment, content: newContent } : comment
+        )
+      )
+    } catch (error) {
+      console.error('댓글 수정 실패', error)
+    }
+  }
+
+  const deleteComment = async (id: string) => {
+    try {
+      await fetch(`https://sp-taskify-api.vercel.app/comments/${id}`, {
+        method: 'DELETE',
+      })
+      setComments((prev) => prev.filter((comment) => comment.id !== id))
+    } catch (error) {
+      console.error('댓글 삭제 실패', error)
+    }
+  }
+
+  useEffect(() => {
+    fetchComments()
+  }, [])
+
+  return (
+    <div className={styles.overlay}>
+      <div className={styles.modalContainer}>
+        <div className={styles.modaltop}>
+          <h1 className={`text-2xl-bold ${styles.cardTitle}`}>{card.title}</h1>
+          <div className={styles.topRightButtons}>
+            <button
+              onClick={() => setIsDropdownOpen((prev) => !prev)}
+              className={styles.buttonbox}
+            >
+              <div className={styles.kebabbutton}>
+                <Image src="/assets/icon/kebab.svg" alt="메뉴" fill />
+              </div>
+            </button>
+            {isDropdownOpen && (
+              <div className={styles.dropdownMenu}>
+                <button
+                  className="block w-full text-left p-2"
+                  onClick={() => onEdit(card)}
+                >
+                  수정하기
+                </button>
+                <button
+                  className="block w-full text-left p-2 text-red-500"
+                  onClick={() => {
+                    if (confirm('정말 삭제하시겠습니까?')) {
+                      onDelete(card.id)
+                    }
+                  }}
+                >
+                  삭제하기
+                </button>
+              </div>
+            )}
+
+            <button onClick={onClose} className={styles.buttonbox}>
+              <div className={styles.closebutton}>
+                <Image src="/assets/icon/close.svg" alt="닫기" fill />
+              </div>
+            </button>
+          </div>
+        </div>
+
+        <section className={styles.assigneebox}>
+          <div className={styles.assigneeInfo}>
+            <span className={styles.assigneespans}>담당자</span>
+            <br />
+            {card.assignee.name}
+          </div>
+          <div className={styles.dueDateInfo}>
+            <span className={styles.assigneespans}>마감일</span>
+            <br />
+            {formatDate(card.dueDate)}
+          </div>
+        </section>
+
+        <div className={`${styles.sectionInfo}`}>
+          <div className={styles.statusTagRow}>
+            <div className={styles.cardStatus}>{card.status}</div>
+            <div className={styles.tagList}>
+              <Image
+                src="/assets/icon/vector-icon.svg"
+                alt="상태와 태그 경계선"
+                width={2}
+                height={20}
+              />
+            </div>
+            {card.tags.map((tag, idx) => (
```

Comment:
(마이너)
유지보수 차원에서 변수명을 축약해서 쓰기보다는 그대로 네이밍하는게 좋을것 같습니다!

```suggestion
            {card.tags.map((tag, index) => (
```

Reply:
확인했습니다!